ebos/eclwriter.hh

```diff
@@ -665,12 +630,8 @@ private:
     }
 
     Simulator& simulator_;
-    std::unique_ptr<EclOutputBlackOilModule<TypeTag>> eclOutputModule_;
+    std::unique_ptr<EclOutputBlackOilModule<TypeTag> > eclOutputModule_;
```

Comment:
This (adding a space between the `>>`) used to be necessary in ancient C++, but in modern C++ it is not and should be avoided.